app/controllers/subscriber/group_controller.rb

```diff
@@ -47,6 +47,7 @@ def update
     def groups_query
       possible = Subscription.new(mailing_list: @mailing_list).possible_groups
       possible.where(search_condition('groups.name', 'parents_groups.name')).
+               without_archived.
```

Comment:
Sind denn archivierte Gruppen nicht standardmässig vom Groups Scope ausgeschlossen? Müssen wir in dem Fall nicht noch an ganz vielen anderen Orten `.without_archived` einfügen?

Reply:
Würde wohl Sinn machen.
Bei den Rollen muss dann der default_scope auch so überschrieben werden dass er sowohl das Archivieren, als auch das Soft-Delete berücksichtigt :+1:

Reply:
Ich mag die Idee doch nicht mehr so.
Ich habs dann nach längerer Zeit hingekriegt, bin aber der Meinung dass wir uns da mehr ins Knie schiessen als sonst was.
z.B. hatte ich dann das Problem dass `group.roles` keine Rollen mehr returned hat, weil diese bei archivierten Gruppen halt auch archiviert sind. Ist aber halt dann komisch, wenn ich versuche bei einer (bewusst) archivierten Gruppe die Rollen zu holen und diese nicht geholt werden weil sie archiviert sind.
Daraufhin kann man dann zwar einen scope auf die `has_many` Relation setzen der den `default_scope` wieder `unscoped` aber ich glaube du siehst da schon mein Problem damit.
Es ist irgendwie doof wenn man die ganze Zeit dran denken muss, ich sehe aber schon gefühlt mehr pain bei einem default_scope, statt dieses bei den relevanten Stellen zu ersetzen.